doc/reference/autoinstall-reference.rst

```diff
@@ -1190,6 +1190,40 @@ Example:
       kernel:
         flavour: hwe
 
+.. _ai-kernel-crash-dumps:
+
+kernel-crash-dumps
+~~~~~~~~~~~~~~~~~~
+
+* **type:** mapping, see below
+* **default:** see below
+* **can be interactive:** no
+
+Toggle kernel crash dumps enablement.
+
+The default value is no configuration, which will result in dynamic enablement as described in the following document (doesn't exist yet).
+
+enabled
+^^^^^^^
+
+**type:** boolean
```

Comment:
Maybe we should document `null` as well. And say that the default for kernel-crash-dumps is:

```yaml
kernel-crash-dumps:
    enabled: null
```

Reply:
Sure. I went back and revised that section a little too.